web/web.go

```diff
@@ -224,9 +225,11 @@ func (h *Handler) Quit() <-chan struct{} {
 	return h.quitCh
 }
 
-// Reload returns the receive-only channel that signals configuration reload requests.
-func (h *Handler) Reload() <-chan struct{} {
-	return h.reloadCh
+// Reload returns two cahnnels:
+//   - receive-only channel that signals configuration reload requests.
+//   - send-only channel with errors happened during configuration reload.
```

Comment:
I'm not sure about this interface, it doesn't indicate when a given reload succeeded.